### DISCUSSION TITLE
Actix - python (or any other language) binding ?

### DISCUSSION THREAD

**Discussion by erikpa1** [actix/actix-web#3186] [Ideas]
Hi, I'm using actix web from time to time in my free time project and I think, it could be kind cool, to have actix wrapper in python (or any other language).

You can create pure python package like flask:

```
app = actix()

@app.get("/api/hello")
def _somefn():
     return "Hello"

app.run_forever("0.0.0.0", "443")
```
I think this could create like the fastest python server ever lived,...

And with good mechanism to create native rust active extensions  you could easily combine python api routes with rust api routes, deploy to any cloud provider as python (using rust packages). A lot of cloud provides support python very well.

Also this approach can minimalize dependency problem with python, because all I need (mostly) will be in rust... so on server there will be python interpreter with a few small packages.

Also another advantage will be, one part of team can fast prototype app to deliver proof of concept and then second part can rewrite performance / security sensitive part to rust, win win for everyone I think.

Basically what I suggest, is same thing people with deno, and next js did with rust for javascript backend world
